searcch_backend/api/common/stats.py

```diff
@@ -0,0 +1,34 @@
+# logic for stat collection
+
+from searcch_backend.api.app import db, config_name
+from searcch_backend.api.common.auth import verify_api_key
+from searcch_backend.models.model import Sessions, StatsRecentViews
+from sqlalchemy import func, asc, desc, sql, and_, or_
+import logging
+
+LOG = logging.getLogger(__name__)
+
+class StatsResource():
+
+    def __init__(self, artifact_id, session_id):
+        self.artifact_id = artifact_id
+        self.session_id = session_id
+
+    def recordView(self):
+
+        query = db.session.query(StatsRecentViews).filter(and_(StatsRecentViews.session_id == self.session_id, StatsRecentViews.artifact_id == self.artifact_id)).first()
```

Comment:
Style thing, but if you can please try to split long lines where possible, as the rest of the code does, that would make things more readable.  e.g. in the above, split at the `.filter` and `.first`.

Reply:
Sure David. I'll do that.